### DISCUSSION TITLE
Is there a reason MultiProvider isn't used?

### DISCUSSION THREAD

**Discussion by KeysGilbert** [deckerst/aves#527] [Q&A]
In the process of trying to understand this codebase. I noticed in the AvesAppState build method, different Providers are initialized one after the other. Wouldn't it be cleaner to use a MultiProvider instance?

**Comment by deckerst** [comment]
No reason! ^^ It probably could be used, but things came to be that way organically, one provider at a time. I won't claim the code is the cleanest. I just try to write things I hope to understand later.
